internal/proxy/usecases/helpers_test.go

```diff
@@ -29,7 +30,12 @@ func (m *mockMetricsRepository) Save(metrics *inputs.MetricsInput) error {
 	return args.Error(0)
 }
 
-func (m *mockMetricsRepository) Count(uuid string) (int, error) {
+func (m *mockMetricsRepository) CountAll(uuid string) (int, error) {
```

Comment:
[pinaillage] `Count` ne voulait pas dire que tu comptais tout à la base ? (ou conflit de nom ?)

Reply:
Yes, I think it's more understandable with `CountAll`